chart/cron-connector/templates/deployment.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    # Original Helm labels v
+    app: {{ template "connector.name" . }}
+    component: cron-connector
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+  name: {{ template "connector.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "connector.name" . }}
```

Comment:
Now this breaks the deployment, have you tested the last change?

Reply:
The change works here are test logs https://travis-ci.com/zeerorg/cron-connector/builds/116125419. But I think I should remove matchlabels.

Reply:
matchLabels is needed to map to template:metadata (which corresponds to pod) but I removed app.kubernetes.io/* from the deployment so it doesn't affect there. I can change the matchLabels and corresponding label keys. The change would look like the diff attached but it would functionally be the same.
```
diff --git a/chart/cron-connector/templates/deployment.yaml b/chart/cron-connector/templates/deployment.yaml
index 18ccb33..5886770 100644
--- a/chart/cron-connector/templates/deployment.yaml
+++ b/chart/cron-connector/templates/deployment.yaml
@@ -16,13 +16,13 @@ spec:
   replicas: 1
   selector:
     matchLabels:
-      app.kubernetes.io/name: {{ template "connector.name" . }}
-      app.kubernetes.io/component: cron-connector
+      name: {{ template "connector.name" . }}
+      component: cron-connector
   template:
     metadata:
       labels:
-        app.kubernetes.io/name: {{ template "connector.name" . }}
-        app.kubernetes.io/component: cron-connector
+        name: {{ template "connector.name" . }}
+        component: cron-connector
     spec:
       containers:
       - name: cron-connector
```

Reply:
Yep that looks great. 👍